app/controllers/api/v1/errors_controller.rb

```diff
@@ -6,6 +6,16 @@ def index
   end
 
   def create
+    @subscriber = Subscriber.where("email = ? and website_id = ?", params['email'], @current_site.id)
+    @error = Issue.where("description = ?", params['message'])
+    if @subscriber.blank?
+      @subscriber = Subscriber.create( email: params['email'], name: params['email'], created_at: Time.now, updated_at: Time.now, website_id: @current_site.id)
+    end
+
+    if @error.blank?
+      @error = Issue.create(description: params['message'], created_at: Time.now, updated_at: Time.now, status: 1, website_id: @current_site.id)
```

Comment:
description ca si mai sus nu isi are locul aici. ce trebuie sa salvezi e restul de informatii. foloseste `current_site.issues.create(..)` si nu mai seta website_id in create... Status nu e nevoie sa il setezi pentru ca are default.